dev/podman/setup-local-env-podman.md

```diff
@@ -0,0 +1,53 @@
+## The following are Podman specific steps used to set up on a MacBook (Intel or Apple Silicon)
+
+### Verify installed tools (install if needed)
+
+```sh
+podman --version
+podman version 5.0.1
+kind version
+kind v0.23.0 go1.22.3 darwin/arm64
+tilt version
+v0.33.12, built 2024-03-28
```

Comment:
Minor nit: I like shell prompts in front of commands (e.g. `$`) to distinguish it from output.

Reply:
Missed that. Thanks!